labgrid/__init__.py

```diff
@@ -5,3 +5,7 @@
 from .factory import target_factory
 from .step import step, steps
 from .stepreporter import StepReporter
+
+from importlib.metadata import version
```

Comment:
Wrong import order: standard imports first, then third-party libraries, then local imports.

Reply:
Also, importlib.metadata is not available in 3.6 and marked as provisional:
https://docs.python.org/3/library/importlib.metadata.html

Reply:
The alternative is too require setuptools at runtime.